src/main/scala/com/webtrends/harness/component/akkahttp/websocket/AkkaHttpWebsocket.scala

```diff
@@ -0,0 +1,123 @@
+package com.webtrends.harness.component.akkahttp.websocket
+
+import akka.actor.{Actor, ActorRef, Props}
+import akka.http.scaladsl.model.Uri
+import akka.http.scaladsl.model.ws.{BinaryMessage, Message, TextMessage}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server._
+import akka.stream.scaladsl.{Flow, Sink, Source}
+import akka.stream.{ActorMaterializer, OverflowStrategy}
+import com.webtrends.harness.app.HActor
+import com.webtrends.harness.command.{BaseCommand, Command, CommandBean}
+import com.webtrends.harness.component.akkahttp.{AkkaHttpCommandResponse, ExternalAkkaHttpRouteContainer}
+
+import scala.concurrent.Future
+
+trait AkkaHttpWebsocket extends BaseCommand with HActor {
+  implicit val materializer = ActorMaterializer(None, None)(context)
```

Comment:
Should make this an `implicit def` and have implementing classes provide the materializer. This will allow our commands to use a single `ActorMaterializer`